src/app/browser/qgsinbuiltdataitemproviders.cpp

```diff
@@ -860,7 +863,8 @@ void QgsFavoritesItemGuiProvider::populateContextMenu( QgsDataItem *item, QMenu
     QString directory = QFileDialog::getExistingDirectory( QgisApp::instance(), tr( "Add Directory to Favorites" ) );
     if ( !directory.isEmpty() )
     {
-      QgisApp::instance()->browserModel()->addFavoriteDirectory( directory );
+      QgisApp::instance()
```

Comment:
Do you know what's happened here?

Reply:
Not investigate yet, but I think it's caused to a max / indentation parameter